staging/src/k8s.io/kubectl/pkg/drain/drain.go

```diff
@@ -241,10 +250,11 @@ func (d *Helper) evictPods(pods []corev1.Pod, policyGroupVersion string, getPodF
 			if err != nil {
 				errors = append(errors, err)
 			}
-		case <-globalTimeoutCh:
```

Comment:
Returning here before we read all results in nothing reading the channel.  This causes goroutines returning from waitForDelete above to hang indefinitely.